de.wagnermartin.Plattenalbum.json

```diff
@@ -0,0 +1,44 @@
+{
+  "app-id": "de.wagnermartin.Plattenalbum",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "plattenalbum",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--filesystem=xdg-music:ro",
+    "--filesystem=xdg-run/mpd",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.FileManager1",
+    "--own-name=org.mpris.MediaPlayer2.Plattenalbum"
```

Comment:
If you can change the name to match app id you can drop this. flatpak by default grants access to `org.mpris.MediaPlayer2.$appid`

Reply:
Thanks, didn't know that. It's fixed now.